js/app.js

```diff
@@ -198,9 +198,10 @@ function tagIsSelected(blogTag) {
 
 
 const snippetTypes = {
-    "media_embed": { "property": "media_item__text.value", "tag": "div", "format": "html" },
-    "code_snippet": { "property": "code", "tag": "pre", "format": "text" },
-    "aside_note": { "property": "note_text", "tag": "aside", "format": "html" }
+    "media_embed": { "property": "media_item__text.value", "tag": "div", "format": "html", "classname": "" },
+    "code_snippet": { "property": "code", "tag": "pre", "format": "text", "classname": "" },
+    "aside_note": { "property": "note_text", "tag": "aside", "format": "html", "classname": "" },
+    "quotation": { "property": "quote", "tag": "aside", "format": "html", "classname": "quotation" }
     };
 function processContentSnippets(blogSummary, data)
```

Comment:
This function now has cyclomatic complexity of (at least) 9 and it might be time to refactor. 

(Measurement can be done here: http://www.lizard.ws/)

Recommendations for cyclomatic complexity vary dramatically, but this is usually my recommendation:

> For most routines, a cyclomatic complexity below 4 is considered good; a cyclomatic complexity between 5 and 7 is considered medium complexity, between 8 and 10 is high complexity, and above that is extreme complexity.

https://www.brandonsavage.net/code-complexity-and-clean-code/

Reply:
Yes, I am not entirely happy with this piece of code but wanted to push it out so your posts with quotations can render for the time being. I will have a think about refactoring. Varied structure of different snippets makes this challenging for me.

Reply:
Refactored:

![image](https://user-images.githubusercontent.com/56796626/75716832-e8ab2900-5cc7-11ea-887b-757d5d3808fb.png)